### PR TITLE
Create blockstreet.yml

### PR DESCRIPTION
**Pull request by nkmlombardi**
Thanks!

### CODE REVIEW

File: _data/projects/blockstreet.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Blockstreet aims to become the go-to source for the explanation of complex topics for users that are new to the blockchain ecosystem.
+site: https://blockstreet.io
+tags:
+- javascript
+- vuejs
+- nodejs
+- cryptocurrency
+- blockchain
+upforgrabs:
+  name: help needed
```
Comment (shiftkey) [inline _data/projects/blockstreet.yml]: This value should be the name of a label assigned to issues that are suitable for new contributors to pick up

File: _data/projects/blockstreet.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/blockstreet/interface/issues
```
Comment (shiftkey) [inline _data/projects/blockstreet.yml]: This value should be the URL to the filtered list of issues associated with the label above

**Comment by nkmlombardi** [comment]
Updated PR to reflect tag label for contributors to pick up.

File: _data/projects/blockstreet.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+desc: Blockstreet aims to become the go-to source for the explanation of complex topics for users that are new to the blockchain ecosystem.
+site: https://blockstreet.io
+tags:
+- javascript
+- vuejs
+- nodejs
+- cryptocurrency
+- blockchain
+upforgrabs:
+  name: Status: Available
```
Comment (shiftkey) [inline _data/projects/blockstreet.yml]: Might have to wrap this in quotes - the second `:` is confusing the YAML parser

File: _data/projects/blockstreet.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/blockstreet/interface/issues
```
Comment (shiftkey) [inline _data/projects/blockstreet.yml]: This should now be `https://github.com/blockstreet/interface/labels/Status%3A%20Available`

**Comment by shiftkey** [comment]
@nkmlombardi thanks!

**Pull request merged**